a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -414,38 +409,39 @@ def execute(self, member, vthunder, pool):
             LOG.info("Error occurred")
 
 
-class CreateHealthMonitorOnVthunder(BaseVThunderTask):
+class CreateHealthMonitorOnVThunder(BaseVThunderTask):
     """ Task to create a healthmonitor and associate it with provided pool. """
 
     def execute(self, vthunder):
         """ Execute create health monitor for master vthunder """
-        # TODO : Length of name of healthmonitor for older vThunder devices
-        axapi_version = acos_client.AXAPI_21 if vthunder.axapi_version == 21 else acos_client.AXAPI_30
         try:
             method = None
             url = None
             expect_code = None
-            # TODO : change this
             name = a10constants.VTHUNDER_UDP_HEARTBEAT
             interval = CONF.a10_health_manager.heartbeat_interval
             timeout = 3
             max_retries = 5
             port = CONF.a10_health_manager.bind_port
             ipv4 = CONF.a10_health_manager.bind_ip
             c = self.client_factory(vthunder)
+            if interval < timeout:
+                LOG.warning(
+                    "Interval should be greater than or equal to timeout(3 seconds). Reverting to default(10 seconds).")
+                interval = 10
             out = c.slb.hm.create(name, openstack_mappings.hm_type(c, 'UDP'),
                                   interval, timeout, max_retries, method, url, expect_code,
                                   port, ipv4)
-            LOG.info("Heath Monitor created successfully.")
-        except Exception as e:
-            LOG.info(str(e))
-        try:
-            c = self.client_factory(vthunder)
-            name = a10constants.HM_SERVER
-            ip_address = CONF.a10_health_manager.udp_server_ip_address
-            health_check = a10constants.VTHUNDER_UDP_HEARTBEAT
-            out = c.slb.server.create(name, ip_address, health_check=health_check)
-            LOG.info("Server created successfully. Enabled health check for health monitor.")
+            LOG.info("Health Monitor created successfully.")
+            try:
+                c = self.client_factory(vthunder)
+                name = a10constants.HM_SERVER
+                ip_address = CONF.a10_health_manager.udp_server_ip_address
+                health_check = a10constants.VTHUNDER_UDP_HEARTBEAT
+                out = c.slb.server.create(name, ip_address, health_check=health_check)
+                LOG.info("Server created successfully. Enabled health check for health monitor.")
+            except Exception as e:
```

Comment:
Why's this all getting wrapped under the health monitor `try-except` block? It's not going to throw an error that it won't catch since the block itself is wrapped in it's own catch all `try-except` block.